instrumentation/hibernate/hibernate-6.0/spring-testing/src/test/java/spring/jpa/SpringJpaTest.java

```diff
@@ -0,0 +1,387 @@
+  static void setUp() {
+    context = new AnnotationConfigApplicationContext(PersistenceConfig.class);
+    repo = context.getBean(CustomerRepository.class);
+  }
+
+  @AfterAll
+  static void tearDown() {
+    if (context != null) {
+      context.close();
+    }
+  }
+
+  @SuppressWarnings("deprecation") // DbIncubatingAttributes.DB_NAME has been deprecated
+  @Test
+  void testCurd() {
+    Customer customer = new Customer("Bob", "Anonymous");
+
+    assertNull(customer.getId());
+    assertFalse(testing.runWithSpan("parent", () -> repo.findAll().iterator().hasNext()));
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("parent")
+                        .hasKind(INTERNAL)
+                        .hasNoParent()
+                        .hasAttributes(Attributes.empty()),
+                span ->
+                    span.hasName("SELECT spring.jpa.Customer")
+                        .hasKind(INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            satisfies(
+                                AttributeKey.stringKey("hibernate.session_id"),
+                                val -> val.isInstanceOf(String.class))),
+                span ->
+                    span.hasName("SELECT test.Customer")
+                        .hasKind(CLIENT)
+                        .hasParent(trace.getSpan(1))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(
+                                DB_SYSTEM, DbIncubatingAttributes.DbSystemIncubatingValues.HSQLDB),
+                            equalTo(maybeStable(DB_NAME), "test"),
+                            equalTo(
+                                DbIncubatingAttributes.DB_USER,
+                                emitStableDatabaseSemconv() ? null : "sa"),
+                            equalTo(
+                                DbIncubatingAttributes.DB_CONNECTION_STRING,
+                                emitStableDatabaseSemconv() ? null : "hsqldb:mem:"),
+                            satisfies(
+                                maybeStable(DB_STATEMENT),
+                                val ->
+                                    val.matches(
+                                        "select ([^.]+)\\.id([^,]*),([^.]+)\\.firstName([^,]*),([^.]+)\\.lastName(.*)from Customer(.*)")),
+                            equalTo(maybeStable(DB_OPERATION), "SELECT"),
+                            equalTo(maybeStable(DB_SQL_TABLE), "Customer")),
+                span ->
+                    span.hasName("Transaction.commit")
+                        .hasKind(INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(
+                                AttributeKey.stringKey("hibernate.session_id"),
```

Comment:
since you already have a static import for `AttributeKey.stringKey` could use just `stringKey`